Helper/GenerateShopperConversionId.php

```diff
@@ -0,0 +1,52 @@
+<?php
+
+namespace Adyen\Payment\Helper;
+
+use Magento\Framework\App\Helper\AbstractHelper;
+use Magento\Framework\App\Helper\Context;
+use Adyen\Payment\Helper\Util\Uuid;
+use Magento\Checkout\Model\Session as CheckoutSession;
+
+class GenerateShopperConversionId extends AbstractHelper
+{
+    const SHOPPER_CONVERSION_ID = 'shopper_conversion_id';
+
+    /**
+     * @var CheckoutSession
+     */
+    private $checkoutSession;
+
+    /**
+     * @param Context $context
+     * @param CheckoutSession $checkoutSession
+     */
+    public function __construct(
+        Context $context,
+        CheckoutSession $checkoutSession,
+    ) {
+        parent::__construct($context);
+        $this->checkoutSession = $checkoutSession;
+    }
+
+    /**
+     * Get or generate a ShopperConversionID
+     *
+     * @return string
+     */
+    public function getShopperConversionId(): string
```

Comment:
We always have a valid Quote object in the places where we need to leverage this method. IMO, injecting `CheckoutSession` is redundant. We can pass `$quote` as an argument to this method to simplify the class.